tools/find_deprecated.py

```diff
@@ -207,8 +219,9 @@ def find_deprecations(file_name: Path) -> list[Deprecation]:
         return decorator_visitor.deprecations
 
 
-if __name__ == "__main__":
-    collection = DeprecationCollection(Path(__file__).joinpath("..", "..", "qiskit").resolve())
+def print_main(directory, pending):
```

Comment:
Without the type hint, I would have assumed that `pending` is a bool.

```suggestion
def print_main(directory: str, pending: str) -> None:
```